### PR TITLE
[10.0][MIG] stock_picking_backorder_strategy

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: stock_picking_backorder_strategy/__manifest__.py [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-    "depends": ["stock",
-                ],
-    "data": ["views/stock_views.xml",
-             ],
-    "demo": [],
-    "license": "AGPL-3",
-    'installable': False,
-    "application": False,
+    'name': 'Picking backordering strategies',
+    'version': '10.0.0.1.0',
```
Comment (LoisRForgeFlow) [inline stock_picking_backorder_strategy/__manifest__.py:7]: isn't `X.0.1.0.0` the default?

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM :+1:

File: stock_picking_backorder_strategy/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply stock_picking_backorder_strategy/__manifest__.py:7]: @lreficent Yes of course. I'll change it

**Comment by Jortolsa-S73** [comment]
In 'no_create' mode, the stock move with the remaining quantity remains pending in the same picking. 
In my opinion that move must be cancelled or not created to avoid problems with extra moves.
stock_picking_backorder

- Use no_create option
- Validate picking with remaining quantity
- The picking still pending with a stock move done and the remaining quantity pending in other stock move
- Validate the picking again
- Odoo creates an extra stock move

@rousseldenis do you agree with the solution of not create the stock move with remaining quantity? Could we help you with the fix? We are in the OCA Code Sprint in Sevilla.

**Review by gurneyalex — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@rousseldenis could you have a look at @Jortolsa comments?

**Comment by rousseldenis** [comment]
@lreficent I will come back soon (few days). I'm on another subject  now.

**Comment by rousseldenis** [comment]
@Jortolsa @lreficent  I've done modifications (+ tests) in another commit.

**Review by Jortolsa-S73 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Are you going to squash a bit your commits before merging, @rousseldenis ?

**Comment by rousseldenis** [comment]
@pedrobaeza Done. I want to keep these two commits.

**Pull request merged**